src/api/public/apidocs-new/components/responses/unknown_command_error.yaml

```
@@ -0,0 +1,19 @@
+description: |
+  Bad request.
+
+  XML Schema used for body validation: [status.xsd](../schema/status.xsd)
+content:
+  application/xml; charset=utf-8:
+    schema:
+      $ref: '../schemas/api_response.yaml'
+    examples:
+      unknown_command_error:
+        value:
+          code: unknown_command
+          summary: "Allowed commands are 'register'."
+        summary: Unknown command error
+      bad_request_error:
+        value:
+          code: err_register_save
+          summary: "Could not save the registration, details: Login is the name of an already existing user and Password can't be blank."
+        summary: Bad Request Error
```

```suggestion
      Bad Request:
        value:
          code: err_register_save
          summary: "Could not save the registration, details: Login is the name of an already existing user and Password can't be blank."
```
Nitpick: same as above.